File: effecientcal.kt
fun main() {
    var num = readLine()?.toIntOrNull()
    var num2 = readLine()?.toIntOrNull() ?: 0
    var operator = readLine()
    if (operator == "*") {
        var result : Int
        result = num!! * num2
        println(result)
    }
    if (operator == "/") {
        var result : Int
        result = num!! / num2
        println(result)
    }
    if (operator == "+") {
        var result : Int
        result = num!! + num2
        println(result)
    }
    if (operator == "-") {
        var result : Int
        result = num!! - num2
        println(result)
    }
}

File: effecientcal.js
function effecientcal(num1, num2) {
    let sum = num1 + num2;
    let difference = num1 - num2;
    let product = num1 * num2;
    let quotient = num1 / num2;
    return {
        sum: sum,
        difference: difference,
        product: product,
        quotient: quotient
    };
}
let result = effecientcal(10, 5); //js dont have input taking without HTML
console.log(result);